Apps/Examples/Examples/All Examples/RestrictCoordinateBoundsExample.swift

```diff
@@ -12,16 +12,19 @@ public class RestrictCoordinateBoundsExample: UIViewController, ExampleProtocol
         let mapView = MapView(frame: view.bounds)
         mapView.autoresizingMask = [.flexibleWidth, .flexibleHeight]
         view.addSubview(mapView)
+        
```

Comment:
nit: New line, this is probably one of SwiftLint errors

Reply:
addressed